.github/workflows/kani.yml

```diff
@@ -15,52 +15,45 @@ jobs:
       - name: Checkout Kani
         uses: actions/checkout@v2
 
-      - name: Install dependencies
-        run: ./scripts/setup/${{ matrix.os }}/install_deps.sh
-
-      - name: Install CBMC
-        run: ./scripts/setup/${{ matrix.os }}/install_cbmc.sh
-
-      - name: Install cbmc-viewer
-        run: ./scripts/setup/install_viewer.sh 2.6
-
-      - name: Install Rust toolchain
-        run: ./scripts/setup/install_rustup.sh
-
-      - name: Update submodules
-        run: |
-          git submodule update --init --depth 1
-
-      - name: Build Kani and Kani Library
-        run: |
-          export RUST_BACKTRACE=1
-          cargo build -p kani-compiler
+      - name: Build Kani
+        uses: ./.github/actions/build
+        with:
+            os: ${{ matrix.os }}
 
       - name: Execute Kani regression
         run: ./scripts/kani-regression.sh
 
+  bookrunner:
+    runs-on: ubuntu-20.04
+    permissions:
+      contents: write
```

Comment:
Github won't let me comment on the darn line, but the OTHER workflow no longer needs these permissions.